sci-misc/boinc/boinc-7.9.3.ebuild

```diff
@@ -0,0 +1,196 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+WX_GTK_VER=3.0-gtk3
+
+inherit autotools eutils gnome2-utils linux-info systemd user versionator wxwidgets
```

Comment:
I was going to say please port from `versionator` to `eapi7-ver` but then it seems that eclass is unused?